bigbluebutton-html5/imports/ui/components/whiteboard/styles.js

```diff
@@ -7,6 +7,9 @@ import {
 import Button from '/imports/ui/components/common/button/component';
 
 const TldrawGlobalStyle = createGlobalStyle`
+  div[data-shape="rectangle"] div div div {
+    width: auto !important;
+  }
```

Comment:
What is this for? I'm worried that it will affect the display of ordinary non-poll labels in rectangle shapes.

Reply:
sometimes the text width is not correct, so adding this makes the label use all available space in the rectangle... it should have no impact on other rectangles

[Screencast from 2023-03-15 09-19-43.webm](https://user-images.githubusercontent.com/3728706/225317792-0b36bb66-5e55-4bff-8ea9-ba9d6ec71027.webm)

[Screencast from 2023-03-15 09-20-40.webm](https://user-images.githubusercontent.com/3728706/225318249-c43f0154-7bfe-4038-a7fb-1ab570af6117.webm)